js/widgets/dialog.js

```diff
@@ -80,12 +80,12 @@ return $.widget( "mobile.dialog", {
 			opts = this.options;
 
 		// Class the markup for dialog styling and wrap interior
-		elem.addClass( "ui-dialog" )
+		elem.addClass( "ui-page-dialog" )
```

Comment:
The dialog widget is deprecated. I just renamed the classes, that's all, so I had to touch this file for the deprecated widget to continue to look OK, but I wasn't intent on implementing the classes option for it. Should I?